xocto/storage/storage.py

```diff
@@ -1179,6 +1184,11 @@ def get_size_in_bytes(self, *, s3_object: S3Object) -> int:
         file_stats = os.stat(filepath)
         return file_stats.st_size
 
+    def get_last_modified(self, *, s3_object: S3Object) -> datetime:
```

Comment:
the return type should be `datetime.datetime`

Reply:
Updated, thank you.